src/app/manage-bookings/models/booking-types.ts

```diff
@@ -1,14 +1,17 @@
 import { FlightModel } from '../../flight-booking/models/flight';
 
-export interface TravelerModel {
-  travelerId: number;
+export interface TravelerRequest {
   name: string;
   address: string;
   phone: string;
   email: string;
   dob: Date;
```

Comment:
can we move this into a models folder inside the shared module

Reply:
yes